crowdkit/aggregation/classification/binary_relevance.py

```diff
@@ -0,0 +1,118 @@
+__all__ = ['BinaryRelevance']
+
+from typing import Dict, List, Union, Any
+
+import attr
+import pandas as pd
+from sklearn.preprocessing import MultiLabelBinarizer
+from . import MajorityVote
+
+from ..base import BaseClassificationAggregator
+from ..utils import clone_aggregator
+
+
+@attr.s
+class BinaryRelevance(BaseClassificationAggregator):
+    r"""Simple aggregation algorithm for multi-label classification.
+
+    Binary Relevance is a straightforward approach for multi-label classification aggregation:
+    each label is treated as a class in binary classification problem and aggregated separately using
+    aggregation algorithms for classification, e.g. Majority Vote or Dawid Skene.
+
+    {% note info %}
+
+    If this method is used for single-label classification, the output of the BinaryRelevance method may differ
+    from the output of the basic aggregator used for its intended purpose, since each class generates a binary
+    classification task, and therefore it is considered separately. For example, some objects may not have labels.
+
+    {% endnote %}
+
+    Args:
+        aggregator: Aggregator instance that will be used for each binary classification. All class parameters
+         will be copied, except for the results of previous fit.
+
+    Examples:
+        >>> import pandas as pd
+        >>> from crowdkit.aggregation import BinaryRelevance, DawidSkene
+        >>> df = pd.DataFrame(
+        >>>     [
+        >>>         ['t1', 'w1', ['house', 'tree']],
+        >>>         ['t1', 'w2', ['house']],
+        >>>         ['t1', 'w3', ['house', 'tree', 'grass']],
+        >>>         ['t2', 'w1', ['car']],
+        >>>         ['t2', 'w2', ['car', 'human']],
+        >>>         ['t2', 'w3', ['train']]
+        >>>     ]
+        >>> )
+        >>> df.columns = ['task', 'worker', 'label']
+        >>> result = BinaryRelevance(DawidSkene(n_iter=10)).fit_predict(df)
+
+    Attributes:
+        labels_ (typing.Optional[pandas.core.series.Series]): Tasks' labels.
+            A pandas.Series indexed by `task` such that `labels.loc[task]`
+            is the tasks' aggregated labels.
+
+        aggregators_ (dict[str, BaseClassificationAggregator]): Labels' aggregators matched to classes.
+            A dictionary that matches aggregators to classes.
+            The key is the class found in the source data,
+            and the value is the aggregator used for this class.
+            The set of keys is all the classes that are in the input data.
+    """
+    aggregator: BaseClassificationAggregator = attr.ib(
```

Comment:
Suggest `base_aggregator` instead of `aggregator`. I would also consider making it private property

Reply:
I agree with renaming, but I don't think it's a good idea to make it private because this is a parameter of the class🤔